app/posts/register-trainee-teachers/2021-08-03-homepage-update-july-21.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Homepage update July 21
+description:
+date: 2021-08-03
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+Changes page heading from the service name (‘Register trainee teachers’) to ’Your trainee teachers’, to:
+
+- reduce confusion with the start page that has the same heading
+- make the purpose of the page a little clearer
+
+Adds the organisation name to the heading, this sets context for the ‘Your’ in ‘Your trainee teachers’ and will for users that belong to more than one organisation.
+
+Makes the heading one size larger for consistency with the rest of the service and gives room in heading hierarchy for sections in the page.
+
+Updates the nav to refer to ‘trainees’ consistently instead of ‘records’. The service currently uses ‘records’ and ‘trainees’ either inconsistently or in a way thats pretty subtle — I personally find ‘record’ to be too general and find it helpful to use ‘trainee’.
+
+Removes the lead-in paragraph, this doesn't need to be seen repeatedly by users.
+
+Styles ‘Add a trainee’ as a link and puts it into the new ‘Draft trainees’ section.
+
+Adds sections:
+
+- Draft trainees
+- Registered trainees
+- Guidance and feedback
+
+This gives a stronger distinction between ‘draft’ and non-draft trainee records. This is consistent with the top nav and helps with prioritise incomplete records as a group.
+
+The three actions in draft are shown as a simple list in part because they are conditional on whether the user has any draft trainees or records imported from Apply. If the user hasn’t created any draft trainees but has records from Apply we'd only show them the ‘See draft trainees imported from Apply’ link because the ‘all draft’ link would do the exact same thing. This will also make prioritisation clearer.
```

Comment:
```suggestion
The three actions in draft are shown as a simple list in part because they are conditional on whether the user has any draft trainees or records imported from Apply. If the user hasn’t created any draft trainees but has records from Apply we’d only show them the ‘See draft trainees imported from Apply’ link because the ‘all draft’ link would do the exact same thing. This will also make prioritisation clearer.
```